homeassistant/components/onewire/config_flow.py

```diff
@@ -273,18 +273,16 @@ async def async_step_configure_device(
             return await self._update_options()
 
         self.current_device, description = self.devices_to_configure.popitem()
-        data_schema: vol.Schema
-        if description.family == "28":
```

Comment:
The family is currently always "28". This will change when more config options are implemented. I am not sure what will be the best way forward here.  Either remove it now, and not have to create any new tests. Or, optionally, add an "else: " statement generating an error which would only trigger on a programming error. This will have to have a test to make sure all cases are handled. (Which I am not sure how to do right now)